src/libraries/System.Net.Sockets/tests/FunctionalTests/Connected.cs

```diff
@@ -0,0 +1,28 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Xunit;
+
+namespace System.Net.Sockets.Tests
+{
+    public class Connected
```

Comment:
The way we currently organize tests, I'm not sure if a separate test class is worth it, shouldn't we just move this test to `Connect.cs`?

Reply:
I'll try to move it there.